Documentation/block.md

```diff
@@ -75,6 +75,51 @@ You should see the wordpress app running.
 
 **NOTE:** When running in a vagrant environment, there will be no external IP address to reach wordpress with.  You will only be able to reach wordpress via the `CLUSTER-IP` from inside the Kubernetes cluster.
 
+## Consume the storage: Toolbox sample
```

Comment:
As discussed, I think we need to streamline the mainline path and remove the clutter that is the manual storage consumption tests.  Manual consumption is only really used for sanity testing or troubleshooting.  The mainline should be only walkthroughs for application/pod consumption of the file/block/object storage, and then we should have a separate page (maybe under the advanced section) for the "manual storage consumption" steps in all their complicated multi-instruction glory :)

Reply:
Yes, good feedback to clearly separate the manual instructions